.github/workflows/ci.yml

```diff
@@ -4,21 +4,23 @@ on: ["pull_request", "push"]
 
 jobs:
 
-  tests:
+  steps:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
 
-    - name: Setup Dafny
-      uses: dafny-lang/setup-dafny-action@v1
-      with:
-        dafny-version: "3.8.1"
-
-    
-    - run: make verify_complete
```

Comment:
We should still verify the file.
Without this, while we know that everything in the file exist in the docs,
and vice versa, we don't know that all that Dafny is valid...

Reply:
See below, we are, just inside the devcontainer now instead.